src/playground/with-layout/chat/chat-custom-message.component.html

```diff
@@ -0,0 +1,33 @@
+<nb-chat title="Nebular Conversational UI" size="large">
+    <nb-chat-message *ngFor="let msg of messages"
+      [type]="msg.type"
+      [message]="msg.text"
+      [reply]="msg.reply"
+      [sender]="msg.user.name"
+      [date]="msg.date"
+      [customMessageData]="msg.customMessageData">
+
+      <div *nbCustomMessage="'link'; let data">
+        <a [href]="data.href">{{ data.label }}</a>
+      </div>
+
+      <div class="button-wrapper" *nbCustomMessage="'button'; let data">
+        <button class="custom-button">{{ data }}</button>
```

Comment:
```suggestion
        <button nbButton class="custom-button">{{ data }}</button>
```
Add `nbButton` directive to add styles.